server/utils/helpers/filterDateRangeHelper.js

```diff
@@ -0,0 +1,24 @@
+const filterDateRangeHelper = (start, end) => {
+  let startDate; let endDate;
+  if (!start && !end) {
+    startDate = new Date('2000-01-01');
+    endDate = new Date();
+  } else if (!end) {
+    startDate = new Date(start);
+    endDate = new Date(`${start}T23:59:59.00Z`);
+  } else if (!start) {
+    startDate = new Date(end);
+    endDate = new Date(`${end}T23:59:59.00Z`);
+  } else {
+    startDate = new Date(start);
+    endDate = new Date(`${end}T23:59:59.00Z`);
+  }
+  return [
```

Comment:
Could you make the output explicit by resolving the tenary operation , so that the returned array is readable?